.github/workflows/unit-tests.yml

```diff
@@ -31,18 +31,7 @@ jobs:
 
     name: Run unit tests
 
-    strategy:
-      matrix:
-        os: [macos-11, macos-12]
-        include:
-          # Skipping tests that are known to be failing on MacOS 12
-          - skipped_tests: |
-              --skip 'BrowserServicesKitTests.TrackerAllowlistReferenceTests' \
-              --skip 'BrowserServicesKitTests.SurrogatesUserScriptsTests' \
-              --skip 'BrowserServicesKitTests.ContentBlockerRulesUserScriptsTests' \
-            os: macos-12
-
-    runs-on: ${{ matrix.os }}
+    runs-on: macos-latest
```

Comment:
We need to specify `macos-12` here, as `macos-latest` still points to `macos-11` 🙄.

https://github.com/actions/runner-images#available-images

Reply:
Great success 🙃